sklearn/experimental/tests/test_enable_hist_gradient_boosting.py

```diff
@@ -2,9 +2,13 @@
 
 import textwrap
 
+import pytest
+
+from sklearn.utils import _IS_WASM
 from sklearn.utils._testing import assert_run_python_script
```

Comment:
Since `assert_run_python_script` is only meant to be used when writing tests, couldn't we make `assert_run_python_script` check for `_IS_WASM` internally and raise `SkipTest("Cannot start subprocess on a WASM runtime.")` directly?

No strong opinion, just a suggestion to reduce redundant boilerplate in our test suite.

Reply:
We could, at the same time, I kind of like the fact that currently the xfail mark is next to the xfailed test function. There is three instances of this I think, so maybe good enough for now?

Also I believe there may be a circular import issue if we try to do this right now, because`_IS_WASM` is defined `sklearn/utils/__init__.py` and reusing it in `sklearn/utils/_testing.py` so some rearrangement would be needed.